src/components/BeaconsTable.tsx

```diff
@@ -121,17 +120,19 @@ export const BeaconsTable: FunctionComponent<IBeaconsTableProps> = ({
       icons={tableIcons}
       columns={[
         {
-          title: "Date",
+          title: "Last Updated Date",
```

Comment:
Could we use sentence case here? 

[GDS](https://www.gov.uk/service-manual/design/writing-for-user-interfaces): 

> Use sentence case everywhere, except for proper nouns.

Reply:
Great spot, absolutely 👍